supertokens_python/recipe/usermetadata/interfaces.py

```diff
@@ -1,23 +1,40 @@
 from abc import ABC, abstractmethod
-from typing import Any, Dict
+from typing import Any, Dict, Literal
```

Comment:
Literal should be imported from `typing_extension`. Please see other files that do this. The reason for this is that the version of `typing` in python 3.7 doesn't have literal, and the min version we support is python 3.7